app/controllers/api/v4/facility_teleconsultations_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V4::FacilityTeleconsultationsController < APIController
+  before_action :set_facility
+  before_action :validate_teleconsultation_facility_belongs_to_users_facility_group
+
+  attr_reader :facility
+
+  def show
+    render json: { teleconsultation_phone_number: teleconsultation_phone_number }
+  end
+
+  private
+
+  def teleconsultation_phone_number
+    "#{facility.teleconsultation_isd_code} #{facility.teleconsultation_phone_number}" if
```

Comment:
Is the space ok? Should we smush it together? I instinctively think we should mush it together, but let's check with Sasikanth on what format the client expects (you may have already done so).

Reply:
I've mushed them together for now, will check with Sasikanth if there's a preference either way.